test/quickstart.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-kubeai-tests}
+
+if kind get clusters | grep -q ${KIND_CLUSTER_NAME}; then
+  echo "Cluster ${KIND_CLUSTER_NAME} already exists.. reusing it"
+  else
+  kind create cluster --name ${KIND_CLUSTER_NAME}
+fi
+
+# Capture PID and run skaffold devin background
+skaffold run --tail --port-forward &
```

Comment:
We should make sure this skaffold run never pushes an image or that the image url for skaffold is overwritten with another non-prod repo. I think `skaffold run` can push by default if the target cluster is a remote one. I know this should be ok for kind - but would want to be explicit to safegaurd against this if we switch the cluster someday.

Reply:
See comment on helm command below for an alternative approach that might closer test the quickstart as it will more completely rely helm for install.

Reply:
Skaffold doesn't push when it detects kind. However, let me add an explicit push false.